module/zfs/metaslab.c

```diff
@@ -286,7 +286,8 @@ static const int max_disabled_ms = 3;
  * Time (in seconds) to respect ms_max_size when the metaslab is not loaded.
  * To avoid 64-bit overflow, don't set above UINT32_MAX.
  */
-static unsigned long zfs_metaslab_max_size_cache_sec = 1 * 60 * 60; /* 1 hour */
+static uint64_t zfs_metaslab_max_size_cache_sec = 1 * 60 * 60;
+/* 1 hour */
```

Comment:
This should be 1 line. I am putting this comment here to remind me to fix it tomorrow when I am at my machine.